data/processed/gtr/total_gtr_projects_stem.yaml

```diff
@@ -0,0 +1,29 @@
+api_doc_url: https://gtr.ukri.org/resources/GtR-2-API-v1.7.4.pdf
+api_type: MySQL
+data_date: 20200222
+description_short: STEM projects led by organisations in the region
+description: Total number of projects led by organisations in the NUTS area in STEM subjects (see aux/gtr_stem_disciplines for the stem disciplines we are considering)
+endpoint_url: https://github.com/nestauk/data_getters#gtr
```

Comment:
I'd add the actual URL and the 2 new auth fields (see https://github.com/nestauk/beis-indicators/pull/146#discussion_r382554546)

Reply:
Could you confirm what you mean by actual URL?

Reply:
I mean the URL of the MySQL instance on AWS

Reply:
Ok - actually in this case it is a flat file stored in AWS so I have linked that. I am not clear about what to see re: `is_public` and `auth_provider` as the data comes originally from a public source (gateway to research) but has been processed and is now stored in a private location (Nesta DAPS). Who is the `auth_provider` in this case, Nesta or UKRI?

Reply:
I think this is about reproducibility: anyone finding this repo should be able to fetch data, run the code, and ultimately verify the methodology, check for errors, propose new approached, etc. so they should be able to get the same data you used. If you used `data_getters` the provider is `Nesta` I think. Hopefully there should be some documentation as to how we create that Mysql DB in the `nestauk/nesta` repo. If not, I think we would have to provide that.